src/main/resources/application-prod.yml

```
@@ -1,18 +1,47 @@
+  secret: ${JWT_SECRET}
+  access-expiration: ${JWT_ACCESS_EXPIRATION:3600000}
+  refresh-expiration: ${JWT_REFRESH_EXPIRATION:604800000}
+file:
+  profile-image-dir: C:/Users/user/profile-images
+  profile-image-url-base: /uploads/profiles/
+cloud:
+  aws:
+    region:
+      static: ${AWS_REGION:us-fake-1}
+    s3:
+      bucket: ${AWS_S3_BUCKET_NAME:fake-bucket-placeholder}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)